frontend/packages/dev-console/src/components/monitoring/dashboard/__tests__/MonitoringDashboardGraph.spec.tsx

```diff
@@ -23,7 +22,7 @@ describe('Monitoring Dashboard graph', () => {
 
   it('should render area graph', () => {
     const wrapper = shallow(<MonitoringDashboardGraph {...monitoringDashboardGraphProps} />);
-    expect(wrapper.find(Area).exists()).toBe(true);
+    expect(wrapper.find(QueryBrowser).props().isStack).toBe(true);
```

Comment:
@vikram-raj  can we extend the test to check `isStack` as well in props for normal series and stacked chart?

Reply:
Here is a test case for render the line graph https://github.com/openshift/console/pull/4072/files#diff-a3f715c19539634ce725a50299562c84R31

Reply:
Thanks @vikram-raj